specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -1773,6 +1829,104 @@
         }
       }
     },
+    "AcsCallParticipantBaseEventData": {
+      "description": "Schema of common properties of participant events",
+      "type": "object",
+      "properties": {
+        "user": {
+          "description": "The communication identifier of the participant",
+          "$ref": "../../../../../communication/data-plane/Common/stable/2023-11-15/common.json#/definitions/CommunicationIdentifierModel"
```

Comment:
Why is this using CommunicationIdentifierModel? this is suppose to be call participant model.
![{D8593D6B-AD6C-4CD9-97B8-6A4B3C3D1D02}](https://github.com/user-attachments/assets/9cee99c6-1e37-4cb8-ab01-cfd5fff6057f)